website/docs/configuring_clients.md

```diff
@@ -51,6 +51,86 @@ Now, be sure you click in `Remote notifications` under `Background Modes` to ena
 
 >![CreateDemoApp](./assets/ios/checkBackgroundModes.png)
 
+### Add the pod dependencies
+
+Close _XCode_ and open a terminal, then change your current directory to the folder where you saved your project.
+You should see a content similar to the image below:
+
+>![CreateDemoApp](./assets/ios/project-folder.png)
+
+from there run
+
+```bash
+pod init
+```
+
+That will create a `Podfile` file in the current folder with the following content:
+
+>```bash
+># Uncomment the next line to define a global platform for your project
+># platform :ios, '9.0'
+>
+>target 'Demo App' do
+>  # Comment the next line if you don't want to use dynamic frameworks
+>  use_frameworks!
+>
+>  # Pods for Demo App
+>
+>  target 'Demo AppTests' do
+>    inherit! :search_paths
+>    # Pods for testing
+>  end
+>
+>  target 'Demo AppUITests' do
+>    # Pods for testing
+>  end
+>end
+>```
+
+Uncomment the platform as suggested and add
+```
+pod 'AeroGearPush-Swift', '~> 3.0.0'
+```
+
+to the `Podfile`:
+
+>```bash
+>platform :ios, '9.0'
+>
+>target 'Demo App' do
+>  # Comment the next line if you don't want to use dynamic frameworks
+>  use_frameworks!
+>
+>  # Pods for Demo App
+>
+>  target 'Demo AppTests' do
+>    inherit! :search_paths
+>    pod 'AeroGearPush-Swift', '~> 3.1.0'
```

Comment:
Hi Massimo,
Here you specify '3.1.0',  but above under 'Uncomment the platform as suggested' you specify '3.0.0'.
which one is correct?

Reply:
3.1.0 is the correct one. However it is not published yet (I'm waiting to have access to the pod).
That mean that, when you will test it, you will have to link the pod locally.

Good catch!